src/main/java/org/glassfish/spec/Artifact.java

```diff
@@ -53,21 +53,32 @@
     /**
      * The Maven SNAPSHOT qualifier.
      */
-    public static final String SNAPSHOT_QUALIFIER = "SNAPSHOT";
+    private static final String SNAPSHOT_QUALIFIER = "-SNAPSHOT";
 
     /**
-     * Strip the SNAPSHOT qualifier from a given qualifier.
-     * @param qualifier the qualifier to process
-     * @return a non SNAPSHOT qualifier
+     * The Release Candidate qualifier.
      */
-    public static String stripSnapshotQualifier(final String qualifier) {
-        if (qualifier != null) {
-            if (qualifier.endsWith("-" + SNAPSHOT_QUALIFIER)) {
-                return qualifier.replace("-" + SNAPSHOT_QUALIFIER, "");
-            }
-            return qualifier;
+    private static final String RC_QUALIFIER = "-RC";
+
+    /**
+     * Strip the SNAPSHOT or RC qualifier from a given version.
+     * @param version the qualifier to process
+     * @return a non SNAPSHOT or RC version
+     */
+    public static String stripSnapshotQualifier(final String version) {
```

Comment:
How about changing the method name to reflect the new behavior ?

Reply:
Yeah was thinking about that, but weighting it against keeping the diff as small as possible. But will make that change now.

Reply:
@romain-grecourt okay, change made